### PR TITLE
add explicit cut for FONLL-E

### PR DESCRIPTION
**Pull request by giacomomagni**
We forgot to to update this FONLL-C cut dependency, to work also for FONLL-E. 

I'd like to update also the `theory.db`.

### CODE REVIEW

File: validphys2/src/validphys/cuts/filters.yaml [hunk 1]
```diff
@@ -388,3 +388,11 @@
...
   IC: True
   rule: "Q2 > 8"
+
+- process_type: DIS_NCP_CH
+  reason: |
+    Missing higher order corrections to Delta F_IC, the piece that needs
+    to be added to the FONLL-E calculation in the case of fitted charm.
+  FNS: FONLL-E
+  IC: True
+  rule: "Q2 > 8"
```
Comment (RoyStegeman) [inline validphys2/src/validphys/cuts/filters.yaml:398]:
```suggestion
  rule: "Q2 > 8"

```

**Review by RoyStegeman — CHANGES_REQUESTED** [review]
Could you order the theories by theoryid?

**Comment by giacomomagni** [comment]
> Could you order the theories by theoryid?

done in 4d957fef21903a79945f9ca45d592e7297605524

**Review by RoyStegeman — APPROVED** [review]
Thanks!

**Pull request merged**